Cargo.toml

```diff
@@ -10,11 +10,14 @@ path = "cargo/lib.rs"
 
 [dependencies]
 cty = "0.2.1"
-term = "0.7.0"
+term = { version = "0.7.0", optional = true }
 
 [build-dependencies]
 pkg-config = "0.3.16"
 
 [features]
+default = ["std"]
+std = ["term"]
+
 intel_64bit_linux_ioctl = []
 intel_64bit_systemv_nasm = []
```

Comment:
To match the configuration options available in the CMake and pal.py configuration interfaces, it might make sense to use a `enable_printers` feature here. Then instead of enabling `std` by default, we could enable `std` and `enable_printers`  for each individual target that logically supports each. So this whole block might look something like this:

```toml
[features]
std = []
enable_printers = ["term"]

intel_64bit_linux_ioctl = ["std", "enable_printers"]
intel_64bit_systemv_nasm = []

# Then new targets in the future could enable whichever logical futures they need:
# intel_64bit_windows_ioctl = ["std", "enable_printers"]
# intel_64bit_ms64_masm = []
# intel_64bit_systemv_rustinline = []
```